src/connector/connection_info.rs

```diff
@@ -119,6 +121,8 @@ impl ConnectionInfo {
             ConnectionInfo::Mssql(url) => url.schema(),
             #[cfg(feature = "sqlite")]
             ConnectionInfo::Sqlite { db_name, .. } => db_name,
+            #[cfg(feature = "sqlite")]
+            ConnectionInfo::InMemorySqlite => panic!("called schema_name() on InMemorySqlite"),
```

Comment:
not too happy about this. Wdyt?

Reply:
Why not `Option<&str>`?

Reply:
We use it a lot in the other variants, it would be painful to handle nulls there. But it's definitely possible.

Reply:
I don't want a library to panic. At least a `Result`.

Reply:
I think there's another way: still keep the attached name, but attach with `:memory:` (so no file). That way we can keep this returning a string.